scripts/make_s2and_mini_dataset.py

```diff
@@ -0,0 +1,141 @@
+import os
+import json
+import pickle
+import numpy as np
+import pandas as pd
+from s2and.consts import CONFIG
+
+DATA_DIR = CONFIG["main_data_dir"]
+
+OUTPUT_DIR = os.path.join(DATA_DIR, "s2and_mini")
+if not os.path.exists(OUTPUT_DIR):
+    os.mkdir(OUTPUT_DIR)
+
+# excluding MEDLINE because it has no clusters
+DATASETS = [
+    "aminer",
+    "arnetminer",
+    "inspire",
+    "kisti",
+    "pubmed",
+    "qian",
+    "zbmath",
+]
+
+# load all of the artifacts of each dataset
+clusters_all = []
+signatures_all = []
+X_all = []
+keys_all = []
+papers_all = []
+for dataset in DATASETS:
+    print()
+    print(f"Loading data from {dataset}...")
+
+    for file_name in os.listdir(os.path.join(DATA_DIR, dataset)):
```

Comment:
What's the reason for filtering in a loop rather than directly opening `f"{dataset}_specter.pickle"` etc?